emsdk.py

```diff
@@ -1280,7 +1280,17 @@ def emscripten_npm_install(tool, directory):
   npm = os.path.join(node_path, 'npm' + ('.cmd' if WINDOWS else ''))
   env = os.environ.copy()
   env["PATH"] = node_path + os.pathsep + env["PATH"]
-  subprocess.check_call([npm, 'install', '--production', '--no-audit'], cwd=directory, env=env)
+  print('Running post-install step: npm install ...')
+  try:
+    subprocess.check_output(
+        [npm, 'install', '--production', '--no-audit'],
+        cwd=directory, stderr=subprocess.STDOUT, env=env,
+        universal_newlines=True)
```

Comment:
the redirection might break colors in the shell, I think?

Reply:
(if so, it makes me wonder if the redirection is worth it)

Reply:
It removes colors from the node output yes.   I'm totally fine with that give that most users should never see anything here.

It also disabled the nice progress bars and stuff like that.